x-pack/plugins/reporting/server/core.ts

```diff
@@ -69,19 +69,17 @@ export class ReportingCore {
   private config?: ReportingConfig; // final config, includes dynamic values based on OS type
   private executing: Set<string>;
 
-  public getStartContract: () => ReportingStart;
+  public getContract: () => ReportingSetup;
```

Comment:
Renamed because we can call this from setup as well - ReportingStart and ReportingSetup are the same interface.